services/basket/src/main/java/org/cryptimeleon/incentive/services/basket/model/Basket.java

```diff
@@ -15,23 +15,46 @@
 @AllArgsConstructor
 public class Basket {
     @ApiModelProperty(value = "${basketModel.basketID}")
-    private UUID basketID;
+    private UUID basketID; // unique identifier for the basket
+
     @ApiModelProperty(value = "${basketModel.items}")
-    private Map<String, Integer> items;
+    private Map<String, Integer> items; // list of items that are contained in the basket
+
     @ApiModelProperty(value = "${basketModel.rewardItems}")
-    private List<String> rewardItems;
+    private List<String> rewardItems; // the list of reward items that the user gets once she redeems the basket
+
     @ApiModelProperty(value = "${basketModel.paid}")
-    private boolean paid;
+    private boolean paid; // whether the basket has already been paid
+
+    /*
+    * Whether the user has already executed the spend and earn requests associated with this basket,
+    * i.e. whether she has already received the points and rewards for it.
+    */
     @ApiModelProperty(value = "${basketModel.redeemed}")
     private boolean redeemed;
+
+    /*
+    * Baskets are locked before they are paid/redeemed
+    * which means that no more items can be added to them.
+    */
     @ApiModelProperty(value = "${basketModel.locked}")
     private boolean locked;
+
+    /*
+    * The first redeem request that the user sent for this basket is stored.
+    * This allows for the user to safely retry redeeming in case of connectivity problems
+    * without allowing for easy double-spending attacks.
+    */
     @ApiModelProperty(value = "${basketModel.redeemRequest}")
     private String redeemRequest;
-    // value must be set manually for serialization
+
     @ApiModelProperty(value = "${basketModel.value}")
-    private long value;
+    private long value; // value must be set manually for serialization
```

Comment:
I don't understand that comment.

Reply:
Is outdated anyway :)

Reply:
*will be  outdated with #178

Reply:
Me neither actually but I feel like it was not by me originally